nessie/multipart_test.go

```diff
@@ -33,7 +33,14 @@ func TestMultipartUpload(t *testing.T) {
 	require.NoError(t, err, "failed to create multipart upload")
 	logger.Info("Created multipart upload request")
 
-	completedParts := uploadMultipartParts(t, logger, resp)
+	parts := make([][]byte, multipartNumberOfParts)
+	var partsConcat []byte
+	for i := 0; i < multipartNumberOfParts; i++ {
+		parts[i] = randstr.Bytes(multipartPartSize)
```

Comment:
Let's vary the size a bit -- 6 MiB is a really round number, and we might miss some padding nastiness somewhere.

```suggestion
		parts[i] = randstr.Bytes(multipartPartSize + 1 + rand.Intn(19))
```

I know that the previous code _also_ used these round numbers; I just think it will be better, seeing as we're here anyway.

Reply:
Anything special about the number 19 here? 
Also, Won't this introduce randomness in tests? i.e, sometimes it passes and sometimes it does not?

Reply:
You're right, 19 is nothing special, and it does introduce randomness.

Regarding randomness: there is indeed a chance of inducing flakiness into the test.  However this is an integration test, flakiness is somewhat expected.  A very reasonable alternative would be to have many more parts (around 20, perhaps 19...) of sizes `multiPartSize + 17 * i` (again, arbitrary numbers).